apps/emqx_bridge_kafka/src/emqx_bridge_kafka_impl_consumer.erl

```diff
@@ -220,10 +220,17 @@ on_stop(ConnectorResId, State) ->
 
 -spec on_get_status(connector_resource_id(), connector_state()) ->
     ?status_connected | ?status_disconnected.
-on_get_status(_ConnectorResId, _State = #{kafka_client_id := ClientID}) ->
+on_get_status(_ConnectorResId, State = #{kafka_client_id := ClientID}) ->
     case brod_sup:find_client(ClientID) of
```

Comment:
would be good to improve brod_supervisor3 to add a find_child gen_call.
when there are a lot of consumer processes, this call makes brod_sup process super busy copying the children list over and over again.

Reply:
second though: this clientid is an atom used to register the process name. maybe no need to lookup.

Reply:
Done, thanks.